### PR TITLE
Allow local development using "skeleton" repos

### PR DESCRIPTION
**Pull request by philipjohn**
This PR switches things around a little so that VIPs can clone their fork of [automattic/vip-skeleton](https://github.com/Automattic/vip-skeleton) into the vagrant in order to more easily develop locally.

It also removes NFS, which isn't compatible with encrypted folders.

### CODE REVIEW

File: Vagrantfile [hunk 1]
```diff
@@ -11,8 +11,10 @@ Vagrant.configure(2) do |config|
   is_windows = RUBY_PLATFORM.downcase.include?("w32");
```
Comment (david-binda) [inline Vagrantfile]: If this variable is no longer being used, it should be removed, imho

**Comment by joshbetz** [comment]
I get that NFS isn't compatible with encrypted folders, but are we sure it's not compatible with encrypted drives? I've had full disk encryption turned on forever and I don't remember having trouble.

**Comment by david-binda** [comment]
@joshbetz The [Vagrant documentation](https://docs.vagrantup.com/v2/synced-folders/nfs.html) is not 100% clear on this:

> Encrypted folders: If you have an encrypted disk, then NFS very often will refuse to export the filesystem. The error message given by NFS is often not clear. One error message seen is <path> does not support NFS. There is no workaround for this other than sharing a directory which isn't encrypted.

The thing that there is no workaround for this seems like a blocker to me, despite I haven't had any issue on Mac OS X with encrypted disk as well.

**Comment by joshbetz** [comment]
Ok, I think it's referring to an encrypted directory rather than the full disk. I think we should add NFS back unless/until we can reproduce problems.

**Comment by philipjohn** [comment]
I've had trouble because my home folder, not the whole disk, is encrypted. So it may well be that full disk encryption isn't an issue.

**Comment by simonwheatley** [comment]
Closing this PR

**Pull request closed without merging**